spec/javascripts/unit/core/pusher_spec.js

```diff
@@ -496,15 +496,16 @@ describe("Pusher", function() {
 
     it("should be sent to a hostname specified in constructor options", function() {
       var pusher = new Pusher("foo", {
-        statsHost: "example.com"
+        statsHost: "example.com",
+        enableStats: true,
       });
       expect(Factory.createTimelineSender).toHaveBeenCalledWith(
         pusher.timeline, { host: "example.com", path: "/timeline/v2/" + timelineTransport }
       );
     });
 
-    it("should not be sent if disableStats option is passed", function() {
-      var pusher = new Pusher("foo", { disableStats: true });
+    it("should not be sent by default", function() {
```

Comment:
Since we consider the `disableStats: false` case important, perhaps we should add a test case for it, by copying the test case for `enableStats: true`?